examples/seq2seq_attn/seq2seq_attn.py

```diff
@@ -101,40 +106,39 @@ def forward(self, batch, mode):
             start_tokens = torch.ones_like(batch['target_length']) * \
```

Comment:
Use `new_full` to create start_tokens, pls refer to PR https://github.com/asyml/texar-pytorch/pull/76